sdk/client.go

```diff
@@ -20,8 +20,9 @@ type Client interface {
 	PostJob(ctx context.Context, headers Headers) (*models.Job, error)
 	PatchJob(ctx context.Context, headers Headers, jobID string, body []PatchOperation) (*RespHeaders, error)
 	PostTask(ctx context.Context, headers Headers, jobID string, taskToCreate models.TaskToCreate) (*RespHeaders, *models.Task, error)
-	GetTask(ctx context.Context, headers Headers, jobID, taskName string) (*RespHeaders, *models.Task, error)
 	GetJob(ctx context.Context, reqheader Headers, jobID string) (*RespHeaders, *models.Job, error)
+	GetJobs(ctx context.Context, reqheader Headers, jobID string) (*RespHeaders, *models.Job, error)
```

Comment:
This needs to pass in the Options:
```suggestion
	GetJobs(ctx context.Context, reqheader Headers, jobID string, options Options) (*RespHeaders, *models.Job, error)
```